docker/web-and-data/docker-entrypoint.sh

```diff
@@ -37,11 +37,11 @@ check_db_connection() {
     eval "$(parse_db_params_from_config_and_command_line $@ | sed 's/^db\./db_/g')"
     POTENTIAL_DB_PARAMS=$@
 
-    while ! mysqladmin ping -s -h$(echo ${db_host} | cut -d: -f1) -u${db_user} -p${db_password};
+    while ! mysqladmin ping -s -h$(echo ${db_host} | cut -d: -f1) -P$(echo ${db_host} | cut -d: -f2) -u${db_user} -p${db_password};
```

Comment:
Thanks so much for your contribution @dtenenba !

It might be nice to handle one more corner case here. It is possible the db_host does not contain a port (the `:` part). Could you maybe separate this out to a variable `$db_port` that's set to 3306 in case `localhost` does not contain `:`?

Reply:
The localdb test might actually be failing correctly, because the host name there doesn't have a `:` in the hostname: https://github.com/cBioPortal/cbioportal-frontend/blob/master/end-to-end-test/local/runtime-config/portal.properties#L5. There are however two subtests in there that are failing now as part of that routine, but it should be able to at least start the tests